woocommerce/woocommerce-ios
# Products: added bottom sheet in Downloadable Files list

Part of #2759 

## Description
As part of Downloadable Files, a feature part of Products M5, we should display a bottom sheet with the following options:
- From device
- From WordPress Media Library
- Enter file URL

In this PR I implemented the bottom sheet, and currently, only the addition of a file URL is working, the other two options will be added in other PRs.

## Changes
- Added a new image from gridicon, representing the WordPress icon + tests.
- Implemented the new `DownloadableFileBottomSheetListSelectorCommand` for selecting a form action for adding a new Donwnloadable File. + Tests
- Implemented `DownloadableFileFormBottomSheetAction`, representing the actions in the downloadable file form bottom sheet.
- `ProductDownloadListViewController`: moved `NSLocalizedString`s strings under the `Localization` enum, and implemented the opening of the bottom sheet when pressing the add button.
- Updated `ProductDownloadListViewModel` for returning all the bottom sheet actions + tests.

## Testing
1. Navigate to a product with Downloadable Files (or enable them from product settings).
2. Tap Downloadable Files row.
3. Press the button "Add File`
4. The bottom sheet should appear.
5. Tap the option "Enter file URL". -> The Add Downloadable File detail should be opened.

## Screenshots
| Dark mode OFF             |  Dark mode ON |
:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone 11 Pro - 2020-10-21 at 18 12 11](https://user-images.githubusercontent.com/495617/96747768-fd5c3e80-13c8-11eb-9606-30440d075243.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-10-21 at 18 12 25](https://user-images.githubusercontent.com/495617/96747772-ff260200-13c8-11eb-9b77-30bfe1cd9c88.png)




Update release notes:
